src/PeoplePage.tsx

```diff
@@ -0,0 +1,48 @@
+import { Loader } from "./components/Loader";
+import { useEffect, useState } from "react";
+import { Person } from "./types";
+import { getPeople } from "./api";
+import { PeopleTable } from "./PeopleTable";
+
+
+export const PeoplePage: React.FC = () => {
+  const [error, setError] = useState('');
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+
+    useEffect(() => {
+      getPeople()
+        .then(data => {
+          setIsLoading(true);
+          setPeople(data);
+        })
+        .catch(() => setError('Something went wrong'))
```

Comment:
In the catch block, you should also catch a possible error, because now the error does not look informative, for example, you can do this:

```js

catch(error) { setIsError(`Something went wrong: ${error.message}`)}
```